debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/FieldSelector.java

```diff
@@ -27,9 +29,14 @@
  */
 @ThreadSafe
 public final class FieldSelector {
+    protected final static Logger LOGGER = LoggerFactory.getLogger(FieldSelector.class);
 
     private static final Pattern DOT = Pattern.compile("\\.");
     private static final Pattern COLON = Pattern.compile(":");
+    private static final String QUALIFIED_FIELD_EXCLUDE_LIST_PATTERN = "<databaseName>.<collectionName>.<fieldName>.<nestedFieldName>";
+    private static final String QUALIFIED_FIELD_RENAMES_PATTERN = "<databaseName>.<collectionName>.<fieldName>.<nestedFieldName>:<newNestedFieldName>";
+
+    private static List<ConfigException> errors;
```

Comment:
Why is this one static? The builder shouldn't interact with the `FieldSelector` class other than instantiating and returning it from the `build()` method.

Reply:
Actually, I was wondering that at the beginning of adding validation to the fields like we have done so far with other fields. But as per Rene's comments on the issue, the entire validation is done inside `FieldSelector` and the exception part was needed to be handled here and in the tests. I am not sure if that was the aim of the issue. Maybe I got it wrong. :/

Reply:
@rk3rn3r ^^^
Could you review the changes and clarify the aim of the issue?